consensus/broadcast/delayedBroadcast.go

```diff
@@ -808,34 +829,34 @@ func (dbb *delayedBlockBroadcaster) getBroadcastDelayForIndex(index int) time.Du
 	return 0
 }
 
-func (dbb *delayedBlockBroadcaster) consensusMessageAlarmExpired(alarmID string) {
+func (dbb *delayedBlockBroadcaster) finalProofAlarmExpired(alarmID string) {
```

Comment:
same as my previous comment, where is the broadcast handled for the final info (before proofs activation)

Reply:
they were specific for equivalent proofs